src/databricks/labs/blueprint/installation.py

```diff
@@ -132,6 +134,10 @@ def check_folder(install_folder: str) -> Installation | None:
             tasks.append(functools.partial(check_folder, service_principal_folder))
         return Threads.strict(f"finding {product} installations", tasks)
 
+    @staticmethod
+    def extension(filename):
```

Comment:
Missing type hints, looks like a redundant methods, typically we use ` Path.suffix`